.github/workflows/CI.yml

```diff
@@ -60,6 +63,7 @@ jobs:
           ppm install linter-eslint
           npm install
           npx atom-package-deps .
+          dir %userprofile%/.atom
```

Comment:
Does this not need to be `.pulsar`?

Reply:
I was trying to see if something got written there 😂